scripts/performance/measure-performance.js

```diff
@@ -0,0 +1,195 @@
+/********************************************************************************
+ * Copyright (C) 2021 STMicroelectronics and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+// @ts-check
+const puppeteer = require('puppeteer');
+const fs = require('fs');
+const resolve = require('path').resolve;
+const path = resolve('./workspace');
+const profilesPath = './profiles/';
+
+const lcp = 'Largest Contentful Paint (LCP)';
+const performanceTag = braceText('Performance');
+
+let name = 'Measurement';
+let url = 'http://localhost:3000/#' + path;
+let folder = 'profile';
+let headless = true;
+let runs = 10;
+
+(async () => {
+    let defaultUrl = true;
+    const args = process.argv.slice(2);
+
+    for (let index = 0; index < args.length; index++) {
+        if (args[index].startsWith('--')) {
+            if (args[index] === '--headful') {
+                headless = false;
+                continue;
+            }
+            let next = args[index + 1];
+            if (next !== undefined && !next.startsWith('--')) {
+                if (args[index] === '--name') {
+                    name = next;
+                    index++;
+                    continue;
+                }
+                if (args[index] === '--url') {
+                    url = next;
+                    defaultUrl = false;
+                    index++;
+                    continue;
+                }
+                if (args[index] === '--folder') {
+                    folder = next;
+                    index++;
+                    continue;
+                }
+                if (args[index] === '--runs') {
+                    runs = parseInt(next);
+                    index++;
+                    continue;
+                }
+            }
+        }
+    }
+
+    if (defaultUrl) getOrCreateFolder(path);
```

Comment:
Curly braces, please (see https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#style). Also, what do we do if the path the user passes in does not exist?